.stylelintrc

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
This file should not be edited and should stay consistent with the generator/all other components:
- https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/.stylelintrc

Reply:
If a style rule is failing, either update the code or add a `/* stylelint-disable (rule) */ in the file where the issue is.

See for example:
https://github.com/AlaskaAirlines/auro-button/blob/e431a34a4e93b47ddd22c2e014c264580c28a116/src/style.scss#L6

Reply:
this file is copied from https://github.com/AlaskaAirlines/auro-input/blob/main/.stylelintrc let me know if there is any other file that we want to copy from.

Reply:
done! :)